docs/customApps.md

```diff
@@ -74,6 +74,30 @@ Before running analysis, make sure that you have `kantra` installed
 1. `./get_latest_kantra_cli.sh` # fetches 'kantra' our analyzer tool and stores it in ../bin
 1. `cd ..`
 
+
+To ensure the analysis uses the necessary custom rules, follow these steps:
+
+1. Custom Rules: Download or create your custom rules and place them in the samples/custom_rules directory.
+2. Ignore Default Rules: The analysis script is set to ignore default Kantra rules. You can review or adjust this in example/analyze.sh if needed.
+---
+<b>Note: </b> There are two different places where analysis is performed and currently only one of them is set to ignore the default Kantra rules:
```

Comment:
Let's relocate this chunk of extra contextual information at the end of the document so we do not confuse folks.